### PR TITLE
Use the same limitrange in -code as in other NSs

### PR DESCRIPTION
**Pull request by alexeykazakov**
See https://issues.redhat.com/browse/CRT-940?focusedCommentId=15836574&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15836574

e2e tests: https://github.com/codeready-toolchain/toolchain-e2e/pull/243

### CODE REVIEW

**Review by rajivnathan — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]
Awesome \o/

File: deploy/templates/nstemplatetiers/basic/ns_code.yaml [hunk 1]
```diff
@@ -67,11 +67,11 @@ objects:
     limits:
     - type: "Container"
       default:
-        cpu: 1000m
-        memory: 512Mi
+        cpu: 500m
```
Comment (ibuziuk) [inline deploy/templates/nstemplatetiers/basic/ns_code.yaml:70]: @alexeykazakov CPU limit should be 1000m, can we apply it across all namespaces?

File: deploy/templates/nstemplatetiers/advanced/ns_code.yaml [hunk 2]
```diff
@@ -67,11 +67,11 @@ objects:
     limits:
     - type: "Container"
       default:
-        cpu: 1000m
-        memory: 512Mi
+        cpu: 500m
```
Comment (ibuziuk) [inline deploy/templates/nstemplatetiers/advanced/ns_code.yaml:70]: should be 1000m, otherwise there will be issues with thiea

**Review by ibuziuk — CHANGES_REQUESTED** [review]
@alexeykazakov I believe we need to opt for higher CPU limit, also could you clarify if those CPU limits were applied on the 'test' tier?

File: deploy/templates/nstemplatetiers/basic/ns_code.yaml (hunk 1 above)
Comment (alexeykazakov) [reply deploy/templates/nstemplatetiers/basic/ns_code.yaml:70]: But you did testing for 500m: https://issues.redhat.com/browse/CRT-940?focusedCommentId=15836574&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15836574

Comment (alexeykazakov) [reply deploy/templates/nstemplatetiers/basic/ns_code.yaml:70]: We could increase it to 1000m though I guess.. The default memory limit allows you to have less than 10 containers anyway. So limiting the number of containers by cpu limits to 10 containers should not change much..
I wish you mentioned that in https://issues.redhat.com/browse/CRT-940?focusedCommentId=15836574&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15836574 before I created the PRs though ;)

**Comment by alexeykazakov** [comment]
> could you clarify if those CPU limits were applied on the 'test' tier?

Yes: https://github.com/codeready-toolchain/host-operator/blob/master/deploy/templates/nstemplatetiers/test/ns_code.yaml#L66-L74
This is what you guys tested.

**Comment by ibuziuk** [comment]
@alexeykazakov I did not test it myself, so probably I need to be provisioned next week against the test tier and verify to be on the safe side. I do suspect that `cpu: 500m` will bring troubles with theia.

File: deploy/templates/nstemplatetiers/basic/ns_code.yaml (hunk 1 above)
Comment (ibuziuk) [reply deploy/templates/nstemplatetiers/basic/ns_code.yaml:70]: Frankly, I was focused on RAM only and missed the CPU part during 'test' tier PR review. 
Need to have yet another round of verification on our end, but I suspect that we might need to stick to 750Mb and 1000m limits across namespaces

Comment (alexeykazakov) [reply deploy/templates/nstemplatetiers/basic/ns_code.yaml]: https://github.com/codeready-toolchain/host-operator/pull/380/commits/4af3c3a19232579eec2de2c8d9923a4608bf5db3

**Comment by ibuziuk** [comment]
@alexeykazakov not sure why the PR was merged - I wanted to verify the new setup properly before rolling it out https://github.com/codeready-toolchain/host-operator/pull/380#issuecomment-786108731

**Comment by MatousJobanek** [comment]
@ibuziuk Check this commit https://github.com/codeready-toolchain/host-operator/commit/4af3c3a19232579eec2de2c8d9923a4608bf5db3 @alexeykazakov increased the cpu limit to 1 core

**Comment by ibuziuk** [comment]
right, seems to work fine, but though we would have more coordination before rolling it out

**Pull request merged**